frontend/cypress/support/component-index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Components App</title>
+  </head>
+  <body>
+    <div data-cy-root></div>
```

Comment:
`data-cy-root` ??? not `data-cy="root"`?

btw (by the way) there is cy.root() command but to get html element.
maybe you don't need root div, but root body?